README.md

```diff
@@ -2,7 +2,11 @@
 [![Build Status](https://travis-ci.org/thoughtbot/french_toast.svg?branch=master)](https://travis-ci.org/thoughtbot/french_toast)
 [![Code Climate](https://codeclimate.com/github/thoughtbot/french_toast/badges/gpa.svg)](https://codeclimate.com/github/thoughtbot/french_toast)
 
-Communicate information about background jobs.
+Communicate information about background jobs seamlessly.
+
+FrenchToast gives users the ability to communicate information based on what the user is able to support. If this means that a user is not online, then they should be emailed. If they don't have javascript, they should be able to refresh the page.
```

Comment:
Can we simplify this language? e.g. “FrenchToast lets applications notify users via the best available channel. …”